Dockerfile

```diff
@@ -1,3 +1,11 @@
+# ------------------
+# Tmp image to precompile
+# ------------------
+FROM badouralix/curl-jq AS precompile
```

Comment:
not the most confidence inducing name 😄

Reply:
yeah, but it's downloaded 1M+ and recently updated. Could have gone for an apt-install but at least it's cached easily